gulp/lib/glob.js

```diff
@@ -1,50 +1,59 @@
-var path = require('path');
+var path = require('path').posix;
```

Comment:
(node v0.12.x) I've tested it on windows, no issues founded, maby it appears on older node/npm versions. Nevertheless I will close this issue.